### PR TITLE
Initial Elasticache blueprint

### PR DESCRIPTION
**Pull request by phobologic**
Only handles Redis ReplicationGroups currently.

### CODE REVIEW

File: stacker/blueprints/elasticache/base.py [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+            },
+            "ParameterGroupFamily": {
+                "type": "String",
+                "description": "The parametergroup family to use, dependent "
+                               "on the engine.",
+                "allowed_values": self.get_parameter_group_family()
+            },
+            "AutomaticFailoverEnabled": {
+                "type": "String",
+                "description": "Set to false to disallow automatic failover "
```
Comment (mhahn) [inline stacker/blueprints/elasticache/base.py]: i had a better comment in mine:
```
Specifies whether a read-only replica will be automatically
promoted to read/write primary if the existing primary fails.
If true, Multi-AZ is enabled for this replication group. If
false, Multi-AZ is disabled for this replication group.
```

File: stacker/blueprints/elasticache/base.py [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+            "EngineVersion": {
+                "type": "String",
+                "description": "Engine version for the Cache Cluster.",
+            },
+            "NotificationTopicArn": {
+                "type": "String",
+                "description": "ARN of the SNS Topic to publish events to.",
+                "default": "",
+            },
+            "NumCacheClusters": {
```
Comment (mhahn) [inline stacker/blueprints/elasticache/base.py]: i would add the comment about the Multi-AZ being enabled:
```
The number of cache clusters this replication group will
initially have. If Multi-AZ is enabled, the value of this 
parameter must be at least 2.
```

**Comment by mhahn** [comment]
should we put this in the new stacker_blueprints repo?

**Comment by phobologic** [comment]
Closing in favor of https://github.com/remind101/stacker_blueprints/pull/3

**Pull request closed without merging**